### ISSUE TITLE
Missing client parameters

### ISSUE THREAD

**Issue by heggi** [Izumi-kun/yii2-longpoll#1]
I'm using longpoll for update news content on page.
In view:
```
\izumi\longpoll\widgets\LongPoll::widget([
    'url' => ['polling'],
    'events' => ['addTranslationBlock'],
    'requestParams' => [
        'id' => $post->id,
        'block_id' => $lastblock,
    ],
    'callback' => 'addTranslationBlock',
]);
```
In controller:
```
  public function actions() {
        return [
            'polling' => [
                'class' => 'izumi\longpoll\LongPollAction',
                'events' => ['addTranslationBlock'],
                'callback' => [$this, 'longPollCallback'],
            ],
        ];
    }

    public function longPollCallback(\izumi\longpoll\Server $server) {
        $id = Yii::$app->request->get('id');
        $block_id = Yii::$app->request->get('block_id');
        if(!$id || !$block_id) throw new \yii\web\NotFoundHttpException();

        $blocks = PostBlock::find()->
            where(['post_id' => $id])->
            andWhere(['>', 'id', $block_id])->
            orderBy(['created' => SORT_ASC, 'order' => SORT_ASC])->
            all();

        foreach($blocks as $block) {
            $server->responseData .= $this->renderPartial('_block.php', [
                'block' => $block,
                'yellow' => true,
            ]);
            $lastBlockId = $block->id;
        }
        
        $server->responseParams = [
            'id' => $id,
            'block_id' => $lastBlockId,
        ];
    }
```
It's do request to server like `polling?id=36&block_id=97&event-addTranslationBlock=17`
It's work fine.
BUT!
If longpoll connection closed by timeout, server returns `{"data":null,"params":{"event-addTranslationBlock":17}}` and the next request is `polling?event-addTranslationBlock=17`.
Additional parameters like id and block_id are missing.

**Comment by heggi** [comment]
Fix:
Server.php run() before foreach:
`$this->responseParams = Yii::$app->request->getQueryParams();`
Within the foreach:
`unset($this->responseParams[$event->getParamName()]);`

**Comment by Izumi-kun** [comment]
Thanks for bug report.
If longpoll connection closed by timeout, client side should send same request. So i think this issue should be fixed in this script: https://github.com/Izumi-kun/jquery-longpoll-client/blob/f9873f16eea255852a9ebcd748173d17d9ab8a8c/src/jquery.longpoll.js#L101.

**Comment by Izumi-kun** [comment]
fixed in 0.4.1
